go.mod

```diff
@@ -16,3 +16,5 @@ require (
 	github.com/posener/complete v1.2.2-0.20190702141536-6ffe496ea953
 	github.com/secure-io/sio-go v0.3.0
 )
+
+replace github.com/minio/minio-go/v6 v6.0.55 => ../minio-go
```

Comment:
can you point to the latest release? @klauspost

Reply:
Yeah, it is broken on purpose until I can verify it.

Reply:
Yes saw it, but not released yet. Not much point in setting it to an inbetween version.